### PR TITLE
(tests) Mark some upgrade tests for locked files FossOnly

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

Mark a few upgrade tests concerning locked files FossOnly for now.

## Motivation and Context

These tests fail inexplicably on teamcity only if CLE is installed, but these failures can't be reproduced locally.

Marking them as FossOnly for now, we'll file follow up issues in CLE to investigate.

## Testing

N/A, just flagging partially broken CLE tests so we can follow up on them later.

### Operating Systems Testing

N/A

## Change Types Made

* [ ] Bug fix (non-breaking change).
* [ ] Feature / Enhancement (non-breaking change).
* [ ] Breaking change (fix or feature that could cause existing functionality to change).
* [ ] Documentation changes.
* [ ] PowerShell code changes.

## Change Checklist

* [ ] Requires a change to the documentation.
* [ ] Documentation has been updated.
* [ ] Tests to cover my changes, have been added.
* [x] All new and existing tests passed?
* [ ] PowerShell code changes: PowerShell v2 compatibility checked?

## Related Issue

N/A

### CODE REVIEW

**Review by gep13 — APPROVED** [review]
LGTM!

File: tests/chocolatey-tests/commands/choco-upgrade.Tests.ps1 [hunk 1]
```diff
@@ -415,16 +427,24 @@ To upgrade a local, or remote file, you may use:
...
             "$env:ChocolateyInstall\lib-bad\$PackageUnderTest\$PackageVersion\$_" | Should -Not -Exist
         }
 
-        It "Outputs a message showing that installation failed." {
-            $Output.Lines | Should -Contain "Chocolatey upgraded 0/1 packages. 1 packages failed."
+        # This does not work as expected when runnning in test kitchen and with Chocolatey Licensed Extension installed.
+        # It is believed to be a problem with test kitchen, or the VM that we are using that is causing the issue.
+        # The result is that versioned backup files of each file in the package is created, instead of the package being
+        # removed. Consider the test partially broken.
+        It "Outputs a message showing that installation failed." -TagFossOnly {
```
Comment (AdmiringWorm) [inline tests/chocolatey-tests/commands/choco-upgrade.Tests.ps1:446]:
```suggestion
        It "Outputs a message showing that installation failed." -Tag FossOnly {

```

**Review by AdmiringWorm — DISMISSED** [review]
Single change needed.

File: tests/chocolatey-tests/commands/choco-upgrade.Tests.ps1 (hunk 1 above)
Comment (gep13) [reply tests/chocolatey-tests/commands/choco-upgrade.Tests.ps1:446]: Great catch!

**Comment by gep13** [comment]
@vexx32 thanks for getting this fixed up!

**Pull request merged**